commands/Demote.js

```diff
@@ -12,7 +12,7 @@ module.exports = class Demote extends Command{
 		});
 	}
 
-	async execute(msg, args){	// eslint-disable-line no-unused-vars
```

Comment:
Please leave this in until args is actually used, else eslint will bitch

Reply:
@Buluphont `args` is used here, first on line 23. I ran ESLint through the whole project on commit so we should be good 👍

Reply:
Oh sick couldn't tell from the PR view. Awesome.